src/scanner/custom-rules/landmark-rule.ts

```diff
@@ -1,15 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-import { RuleConfiguration } from '../iruleresults';
-
-const landmarkCheckId: string = 'unique-landmark';
-
-export const landmarkConfiguration: RuleConfiguration = {
-    checks: [],
-    rule: {
-        id: 'main-landmark',
-        selector: '[role=main], main',
```

Comment:
In the rule we have removed, we check for the HTML technique too instead of just role. It looks like `doesResultHaveMainRole` doesn't have that - are we still picking up on <main> elements?

Reply:
Good question! Yes, we are (the gifs in the description are testing against `<main>` and `<banner>` elements rather than explicit role attributes) - the information in the `role` checkResult data is populated in the `unique-landmark.ts` custom rule implementation, which uses an explicit role element if present but falls back to using `axe.commons.aria.implicitRole(element)` when an explicit role attribute is not present.

Reply:
Adding a comment to clarify